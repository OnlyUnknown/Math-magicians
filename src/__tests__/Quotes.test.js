import renderer from 'react-test-renderer';
import Quote from '../components/Quotes';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom'


describe('Calculate.js File Testing', () => {
    test("Render by Testing library", () => {
        render(<Quote/>); 
        const QuoteContent = screen.getByTestId("QuoteContent");
        expect(QuoteContent).toBeInTheDocument()
        })


test('renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
});