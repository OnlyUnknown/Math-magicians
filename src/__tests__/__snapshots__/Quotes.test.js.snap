// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculate.js File Testing renders correctly 1`] = `
<div
  className="quote"
  data-testid="QuoteContent"
>
  <p>
    -
  </p>
</div>
`;
